quisp/modules/BellStateAnalyzer.cc

```diff
@@ -332,6 +339,25 @@ void BellStateAnalyzer::finish() {
   std::cout << "Success = " << DEBUG_success << "\n";
   std::cout << "darkcount_count_left = " << DEBUG_darkcount_left << ", darkcount_count_right =" << DEBUG_darkcount_right << ", darkcount_count_both = " << DEBUG_darkcount_both
             << "\n";
+  std::cout << "total BSA performance"
+            << "\n";
+  // filename for recoding bsa performance
+  // std::string file_name = BSA_perf_output_filename;
+  // int file_size = file_name.size();
+  // create file
+  std::string a = std::to_string(duration);
```

Comment:
please don't give a meaningless name such as `std::string a` for readability.
in this case, we can conbine L348 and L349 like this
```
   std::string file_name = "num_trials" + std::to_string(duration); 
```

Reply:
I got it. Thank you for notifying.